.eslintrc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   env: {
     node: true
   },
-  extends: ["plugin:vue/essential", "@vue/prettier"],
+  extends: ["standard", "plugin:vue/essential", "@vue/prettier"],
```

Comment:
Standardjs conflicts with Prettier,
so let's just replace `@vue/prettier` with `@vue/standard` (https://github.com/vuejs/eslint-config-standard)

Reply:
I've updated the code and replaced @vue/prettier with @vue/standard